src/styles/colors.js

```diff
@@ -47,7 +47,8 @@ let base = {
   transparent: 'transparent',
   white: '#FFFFFF', // '255, 255, 255'
   yellow: '#FFD657', // '255, 214, 87'
-  yellowOrange: '#FFC400', // '255, 178, 0'
+  yellowFavorite: '#FFB200', // '255, 178, 0'
```

Comment:
I noticed that the existing "yellowOrange" hex value had the wrong RGB values commented next to it. The yellowFavorite is more orange-y than the yellowOrange, but I kept the name like this for now. @christianbaroni pls rename if needed.